.github/workflows/tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * 0' # weekly
```

Comment:
is this needed? (does this mean it'll retest weekly?)  I have seen this in other folks' work too and don't see why it is here.

Reply:
this was copied directly from kitchen; I'll take it out